Formula/cava.rb

```diff
@@ -0,0 +1,41 @@
+class Cava < Formula
+  desc "Console-based Audio Visualizer for ALSA"
+  homepage "https://github.com/karlstav/cava"
+  url "https://github.com/karlstav/cava/archive/0.7.4.tar.gz"
+  sha256 "fefd3cc04d41b03ca416630cafadbfda6c75e2ca0869da1f03963dcb13e1ecb7"
+  license "MIT"
+  head "https://github.com/karlstav/cava.git", branch: "master"
+
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "fftw"
+  depends_on "iniparser"
+  depends_on "portaudio"
+
+  uses_from_macos "ncurses"
+
+  def install
+    # force autogen.sh to look for and use our glibtoolize
+    inreplace "autogen.sh", "libtoolize", "glibtoolize"
+    # correct Makefile.am with hardcoded libpath
+    inreplace "Makefile.am", "/usr/local", HOMEBREW_PREFIX
```

Comment:
Looks like this `inreplace` broke the build on Intel too.

Reply:
Well, I did some digging:
It builds without issues locally when I set automake and libtool as "depends_on" without ":build"...
But I do not know why that is this way.

Shouldn't it work when just building?

Reply:
Does it stop working locally when you make them `:build` dependencies?

Reply:
You can make them `:build` dependencies if you also add a build dependency on `autoconf`.

Reply:
ahh okay. :build dependencies have to list piece by piece... I thought since automake has a dependency on autoconf, all dependencies are there...

Reply:
Can we report the hardcoded `/usr/local` upstream? The `libtoolize` `inreplace` is fine, but they must support configurable prefixes.

Reply:
I ask them.
The /usr/local is not referring to the prefix. This is set correctly.
It refers to the LDFLAGS.

Reply:
Yes, it doesn't refer to the prefix, but this also causes build failures on systems that don't have `/usr/local`.